Dockerfile

```
@@ -0,0 +1,57 @@
+#-------------------------------#
+# Development/build environment #
+#-------------------------------#
+
+FROM  debian:11.2 as taste-dev-env
+
+# Meta
+LABEL VENDOR="N7 Space"
+LABEL ARTEFACT="TASTE Development Environment"
+LABEL DESCRIPTION="Build environment for additional TASTE tools"
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DOTNET_CLI_TELEMETRY_OPTOUT=true
+ENV PATH="/asn1scc/asn1scc/bin/Debug/net5.0/:${PATH}"
+
+# Setup apt dependencies
+RUN apt-get update && apt-get install -y \
+    apt-transport-https \
+    git \
+    libglu1-mesa-dev \
+    make \
+    openjdk-11-jre \
+    python3-pexpect \
+    python3-pip \
+    python3-pygraphviz \
+    python3-singledispatch \
+    python3-stringtemplate3 \
+    spin \
+    wget
```

```suggestion
    wget \
    && rm -rf /var/lib/apt/lists/*
```
to reduce image size

---

Is it worth it with an image over 2GB in size?

---

it can be ~400MB and it stacks with each layer that uses apt update (it usually is around 100MB though)

---

ack